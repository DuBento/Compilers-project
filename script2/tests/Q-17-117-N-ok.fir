int *fir() -> 0 @ {
  float pi = 3.14;
  writeln pi;
}
{
  float pi = 3.141;
  writeln pi;
}
>> {
  float pi = 3.1415;
  writeln pi;
}
