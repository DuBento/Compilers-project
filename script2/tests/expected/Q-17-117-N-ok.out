3.14
3.141
3.1415
